### PR TITLE
Version 2.3 Release Candidate

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses # (issue)

pysat v2.3 Release

## Type of change

Please delete options that are not relevant.

- Version Release

# How Has This Been Tested?

Unit Tests

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zeondo.json
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (https://pysat.readthedocs.io/en/v2-3-rc/)
- [x] Upload the test distribution to the testPyPi server
- [x] Contacted developers and users about the future release
- [ ] Performed integration tests with downstream packages (include list of these packages)
- [x] Create a wiki page with the release text (https://github.com/pysat/pysat/wiki/v2.3.0-Release-Text)
- [x] After approval, confirm all changes from the RC pull are listed in the wiki release text.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,42 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.3.0] - 2021-XX-XX
```
Comment (rstoneback) [inline CHANGELOG.md:5]:
```suggestion
## [2.3.0] - 2021-04-01
```

**Comment by aburrell** [comment]
Seems like it would be simpler to re-do this pull after #754 is done.  Recommend closing it.

**Comment by rstoneback** [comment]
> Seems like it would be simpler to re-do this pull after #754 is done. Recommend closing it.

I wanted to get some of the pre-work in setting up this pull done last night. Trying to keep time available today for releases. I pulled in the latest changes.

**Comment by rstoneback** [comment]
Uploaded to test pypi

**Comment by rstoneback** [comment]
pip install -i https://test.pypi.org/simple/ pysat==2.3.0rc1

**Comment by rstoneback** [comment]
Updated the links to point to `main-2` and updated to travis.com rather than .org.

**Review by jklenzing — APPROVED** [review]
Looks good to me.  @aburrell?

**Review by aburrell — APPROVED** [review]

**Comment by rstoneback** [comment]
GitHub says I can't officially approve via the button, so, Approved.

**Comment by aburrell** [comment]
Yeah, you can't approve since you're the creator!

**Pull request merged**